src/msw/toolbar.cpp

```diff
@@ -581,11 +581,11 @@ wxSize wxToolBar::DoGetBestSize() const
     wxSize sizeBest;
     if ( IsVertical() )
     {
-        sizeBest.x = sizeTool.x + 2 * ::GetSystemMetrics(SM_CXBORDER);
+        sizeBest.x = sizeTool.x + 2 * wxGetSystemMetrics(SM_CXBORDER);
```

Comment:
Why don't we use either `this` or `m_parent` here (and elsewhere in this file) as window parameter?

Reply:
Thanks for noticing. Forgot to add them. This code was added after the pm-dpi PR.